cyraft/node.py

```diff
@@ -175,7 +175,11 @@ async def _start_election(self) -> None:
         for remote_node in self.cluster:
             if remote_node._node.id != self._node.id:
                 _logger.info("Node ID: %d -- Sending request vote to node %d", self._node.id, remote_node._node.id)
-                response = await remote_node.serve_request_vote_impl(request, self._node.id)
+                # response = await remote_node._serve_request_vote_impl(request, self._node.id)
+                client_node = self._node.make_client(
+                    sirius_cyber_corp.RequestVote_1, remote_node._node.id, "request_vote"
+                )
```

Comment:
This works but is inefficient because to open a client the library has to organize a lot of stuff internally, like open a socket, start a worker task, etc. It is preferable to keep your clients stored away somewhere ready to use, as you described in the OP comment.